app/_hooks/useCoordinate.ts

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from 'react';
+
+const useCoordinate = (address: string) => {
+  const [coords, setCoords] = useState([0, 0]);
+
+  useEffect(() => {
+    window.kakao.maps.load(() => {
+      const geocoder = new window.kakao.maps.services.Geocoder();
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      geocoder.addressSearch(address, function (result: any, status: any) {
```

Comment:
타입을 정의해주시면 사용하는 입장에서 편할 것 같네요..!
`result`는 정보가 많아서 귀찮긴 하네요..
[addressSearch 문서](https://apis.map.kakao.com/web/documentation/#services_Geocoder_addressSearch)
[result 타입 문서](https://developers.kakao.com/docs/latest/ko/local/dev-guide#address-coord-response-body-document)
[status 타입 문서](https://apis.map.kakao.com/web/documentation/#services_Status)

```suggestion
      geocoder.addressSearch(address, function (result: 많아서 생략..., status: 'OK' | 'ZERO_RESULT' | 'ERROR') {
```

Reply:
수정하겠습니다.